plugins/adb/_adb

```diff
@@ -48,8 +48,8 @@ _arguments \
 
 case "$state" in
 	specify_device)
-	_values  'devices' $(adb devices|awk 'NR>1&& $1 ~ /^[a-zA-Z0-9].*$/ \
-		{printf "%s[Device_%d:%s] ",$1,++i,$2 }')
+	_values  'devices' $(adb devices -l|awk 'NR>1&& $1 ~ /^[a-zA-Z0-9].*$/ \
+		{printf "%s[%s] ",$1,$6 }')
```

Comment:
Thanks for your update! However, some devices doesn't have model:
```
21e718b2               device usb:336592896X product:OnePlus model:ONE_E1003 device:OnePlus
emulator-5554          device product:sdk_gphone_x86 model:Android_SDK_built_for_x86 device:generic_x86
```

But, for those cases `$NF` works well:
```diff
-               {printf "%s[%s] ",$1,$6 }')
+               {printf "%s[%s] ",$1,$NF }')
```

Reply:
fixed here~ https://github.com/robbyrussell/oh-my-zsh/pull/6489